components/profile-picture/ProfilePicture.tsx

```diff
@@ -0,0 +1,102 @@
+import { StatusBar } from "expo-status-bar";
+import React, { useEffect, useState } from "react";
+import { StyleSheet, View, Image, Pressable, Animated } from "react-native";
+import { LinearGradient } from "expo-linear-gradient";
+import * as Animatable from "react-native-animatable";
+
+type ProfilePictureProps = {
+  imageSize: number;
+  borderColor: string[];
+  openStories: () => void;
+};
+
+const calculateGradientSize = (imageSize: number): number =>
+  imageSize < 60 ? imageSize + 5 : imageSize + 12;
+
+const calculateInnerCircleSize = (imageSize: number): number =>
+  imageSize < 60 ? imageSize + 2 : imageSize + 7;
+
+const ProfilePicture = ({
+  imageSize,
+  borderColor,
+}: ProfilePictureProps): JSX.Element => {
+  const [isPressing, setIsPressing] = useState(false);
+  const [scale, setScale] = useState(new Animated.Value(isPressing ? 0.92 : 1));
```

Comment:
`useState` receives an `initialState` as an argument. `isPressing` is always `false` initially. 
```suggestion
  const [scale, setScale] = useState(new Animated.Value(1));
```